README.md

```diff
@@ -119,6 +123,55 @@ Serve files relative to `path`.
 Byte offset at which the stream starts, defaults to 0. The start is inclusive,
 meaning `start: 2` will include the 3rd byte in the stream.
 
+##### transform
+
+A function which returns a stream which can be used to transform the data.
+If this option is passed it will change the defaults of `etag` and `lastModified`
```

Comment:
This wording is misleading; if the `transform` option is passed, but as `transform: undefined`, I assume this will not default those to `false`, but the wording says it will as long as the option is _passed_ (without regard to what value is passed).